polaris-service/src/test/java/org/apache/polaris/service/admin/PolarisServiceImplIntegrationTest.java

```diff
@@ -1775,4 +1790,479 @@ public void testAssignListAndRevokeCatalogRoles() {
       assertThat(response).returns(204, Response::getStatus);
     }
   }
+
+  @Test
+  public void testCatalogAdminGrantAndRevokeCatalogRoles() {
+    // Create a PrincipalRole and a new catalog. Grant the catalog_admin role to the new principal
+    // role
+    String principalRoleName = "mypr33";
```

Comment:
Is this just to avoid conflicts across tests? If that's true, we might want to harden this against multiple implementations of `PolarisServiceImplIntegrationTest` running. Can we just randomize it until we have better isolation across tests?

Reply:
I actually did fix this more correctly by adding the logic to the teardown method. Previously things weren't getting deleted because the catalogs weren't empty, but now they're emptied before everything else gets deleted.